x/programs/simulator/src/lib.rs

```diff
@@ -1,12 +1,35 @@
-#![allow(non_upper_case_globals)]
-#![allow(non_camel_case_types)]
-#![allow(non_snake_case)]
-// include the generated bindings
-// reference: https://rust-lang.github.io/rust-bindgen/tutorial-3.html
-include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+mod bindings {
+    #![allow(non_upper_case_globals)]
+    #![allow(non_camel_case_types)]
+    #![allow(non_snake_case)]
+    #![allow(dead_code)]
```

Comment:
why allow dead_code?

Reply:
Soooo much of the generated code isn't actually used. You don't notice when it's in `lib` because it's all marked `pub`, but when you put it in a private mod, it all shows up as dead-code